_docs/geyser/setup.md

```diff
@@ -6,31 +6,52 @@ permalink: /geyser/setup/
 
 # Setup
 
-Bedrock clients will join through Geyser and it will handle all the packet translations. There are six different versions of Geyser: Geyser for Spigot (works on derivatives such as Paper), Geyser for BungeeCord (also works on Waterfall), Geyser for Velocity, Geyser for Sponge, Geyser for Fabric (does not support client required mods, and more info about that can be found [here](/other/geyser-fabric/)) and Geyser Standalone. The first five versions run as plugins/mods and can be installed directly onto the server. The standalone version can be used in a similar way, except you run it separately.
+## Overview
+
+Bedrock clients will join through Geyser, and it will handle all the packet translations. There are six different versions of Geyser: 
+1. Geyser for Spigot (works on derivatives such as Paper),
+2. Geyser for BungeeCord (also works on Waterfall), 
+3. Geyser for Velocity,
+4. Geyser for Sponge, 
+5. Geyser for Fabric (does not support client required mods, and more info about that can be found [here](/other/geyser-fabric/)),
+6. Geyser Standalone (Standalone Setup instructions are [here](/geyser/standalone/)).
+
+The first five versions run as plugins/mods and can be installed directly onto the server. The standalone version can be used in a similar way, except you run it separately.
 
 If you are running a server, it is highly recommended you use one of the plugin versions, and if you want to join a server that does not have Geyser installed, you can run the standalone version. If you use Pterodactyl Panel we have an egg for the standalone version, please see [here](/geyser/faq/#can-i-use-geyser-with-pterodactyl-panel) for more information.
 
-If you're still having problems with Geyser not working or giving you an "Unable to connect to world" error, see the [Common Issues](/geyser/common-issues/) page.\
-For more information, take a look at the [Understanding the Config](/geyser/understanding-the-config/) page, and the [FAQ](/geyser/faq/) page.\
-And if you still have questions, feel free to join the [Discord](https://discord.geysermc.org) if you haven't already.
```

Comment:
Why was this removed?

Reply:
It's not removed, i just moved it to another section - i thought it would make more sense to have this after the setup guide, not infront of it

Reply:
I meant to delete this comment, but it disappeared from the review. Thanks GitHub!